### PR TITLE
Clean up the code

### PR DESCRIPTION
**Pull request by awoo-civ**
Compiles.

### CODE REVIEW

File: src/main/java/com/untamedears/JukeAlert/command/commands/JaToggleLeversCommand.java [hunk 1]
```diff
@@ -15,68 +15,70 @@
...
-                sender.sendMessage(ChatColor.RED + "You do not own any snitches nearby or do not have permission to modify them!");
-                return false;
-            }
-            
-        } else {
-            sender.sendMessage(ChatColor.RED + "You do not own any snitches nearby!");
-            return false;
-        }
-    }
+
```
Comment (ProgrammerDan) [inline src/main/java/com/untamedears/JukeAlert/command/commands/JaToggleLeversCommand.java:20]: Something funny here; three tabs on a Constructor?

**Comment by ProgrammerDan** [comment]
Besides that one QQ looks good

File: src/main/java/com/untamedears/JukeAlert/command/commands/JaToggleLeversCommand.java (hunk 1 above)
Comment (awoo-civ) [reply src/main/java/com/untamedears/JukeAlert/command/commands/JaToggleLeversCommand.java:20]: How'd you notice that... 
Yea looks like it was double indented in the original, I didn't notice.

Comment (ProgrammerDan) [reply src/main/java/com/untamedears/JukeAlert/command/commands/JaToggleLeversCommand.java:20]: I review manually every PR, usually every line, it's why things are a little slower ;P

**Pull request merged**